libs/repositories/src/UsdRepository/UsdRepositoryFallback.ts

```diff
@@ -0,0 +1,38 @@
+import { injectable } from 'inversify';
+import { PricePoint, PriceStrategy, UsdRepository } from './UsdRepository';
+
+@injectable()
+export class UsdRepositoryFallback implements UsdRepository {
+  constructor(private usdRepositories: UsdRepository[]) {}
+
+  async getUsdPrice(
+    chainId: number,
+    tokenAddress: string
+  ): Promise<number | null> {
+    for (const usdRepository of this.usdRepositories) {
+      const price = await usdRepository.getUsdPrice(chainId, tokenAddress);
+      if (price !== null) {
+        return price;
+      }
+    }
```

Comment:
Should we have a way to avoid hammering repositories that fail, for example due to rate limiting and retrying it later?
A retry strategy.

Reply:
Good point, but I would not do it for the rety. I would do it more for the case where we get a `null`, which is most likely a (I don't know this token)

I could iterate, but let me first implement a second repository for getting the USD from our orderbook API

Reply:
https://github.com/cowprotocol/bff/issues/54